src/classic/clvm/__type_compatibility__.rs

```diff
@@ -25,7 +25,7 @@ pub fn char_to_string(ch: char) -> String {
     }
 }
 
-pub fn vec_to_string(r: &Vec<u8>) -> String {
+pub fn vec_to_string(r: &[u8]) -> String {
     return String::from_utf8_lossy(r).as_ref().to_string();
```

Comment:
for now, encoding is nominally utf-8 but there's no special processing for non-utf8 and i'm being lenient here simply in case somebody has used another encoding somewhere.  This is the classic compiler so hopefully we'll deprecate it eventually.